libnmstate/validator.py

```diff
@@ -401,3 +415,36 @@ def _assert_bond_config_is_legal(iface_state, original_iface_state):
                 "Bond option arp_interval is conflicting with miimon, "
                 "please disable one of them by setting to 0"
             )
+
+
+def _validate_slave_ip_config(original_desired_state):
+    """
+    When an interface is created, the slaves/ports cannot have
+    their IP configuration enabled
+    """
+    for iface_state in original_desired_state.interfaces.values():
+        iface_type = iface_state.get(Interface.TYPE)
+        slaves = []
+        if iface_type in SLAVE_GETTERS.keys():
+            slaves = SLAVE_GETTERS[iface_type](iface_state)
+        if slaves:
+            for slave_iface_name in slaves:
+                slave_state = original_desired_state.interfaces.get(
+                    slave_iface_name, {}
+                )
+                if (
+                    slave_state.get(Interface.TYPE)
+                    != InterfaceType.OVS_INTERFACE
+                ):
+                    ipv4_state = slave_state.get(Interface.IPV4, {}).get(
+                        InterfaceIP.ENABLED
+                    )
```

Comment:
Please change it to `InterfaceIP.DISABLED`. We don't want user got `NmstateValueError` when they never defined IP stack on slaves.

Reply:
I havent seen any DISABLED property in `libnmstate/schema.py`. InterfaceIP has the ENABLED property, which can be true or false.

Reply:
You are correct. The code looks good for this part.